build_tools/cmake/test_riscv.sh

```diff
@@ -38,6 +38,8 @@ declare -a label_exclude_args=(
   "^vulkan_uses_vk_khr_shader_float16_int8$"
   "^requires-filesystem$"
   "^requires-dtz$"
+  # Tests labelled heavy-compute can cause timeouts when run in emulator.
+  "^heavy-compute$"
```

Comment:
How about reusing the more standard `size` or `timeout` labels from Bazel?
https://bazel.build/reference/test-encyclopedia#role-test-runner
https://bazel.build/reference/be/common-definitions

(prior discussion here: https://github.com/openxla/iree/issues/5192)

Reply:
I kind of thought of that, but this is one of places where Bazel hardcodes Google server-centric assumptions. `size` values map to specific `timeout` values, implying that we have in the BUILD file some knowledge of the order of magnitude of performance of the test host. It would be technically possible to have a `select` selecting the right timeout for each build configuration, but that would be heavy, nobody is doing that, and as a result, everyone is writing BUILD files with size/timeout settings that hardcode assumptions about test machines.

The other problem is that the established categories are too large for our purposes here. The default `size` of `medium` maps to a 5 minute timeout. That would allow tests that would definitely take too long on emulator. Actually, even without the emulator concern, targeting only normal CPUs, we generally don't want each test to take up to 5 minutes.

Then we could consider filtering only tests with only a smaller `size` (or `timeout`) than the default. But then we would be switching to "run most tests, just opt out of some" to "run few tests, on opt-in basis". That would be a big departure and apparently not what we want. Also, even if we did that, the smallest available category `size=small` is still too large for our purposes (1 minute on normal CPU, still forever on emulator).

Another problem is that `size=***`, `timeout=***` are not that easy to filter in regexes, unless we filter only specific values (conceivable for `size`, less so for `timeout`).

Reply:
"heavy-compute" doesn't different enough from "size" or "timeout"... maybe just add a "noriscv" label instead?

Reply:
Sure let's do that.  I was thinking about generalization but it's not a bad idea to defer generalization until there is more than 1 sample.